lib/Doctrine/ORM/Tools/Console/Command/ClearCache/MetadataCommand.php

```diff
@@ -106,6 +107,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $message = ($result) ? 'Successfully flushed cache entries.' : $message;
         }
 
-        $output->writeln($message);
+        if (!$result) {
```

Comment:
This does not follow our code standards (there must be a space before and after the negation operator)